lib/diaspora_federation.rb

```diff
@@ -74,6 +75,18 @@ class << self
     #   @param [String] value path to certificate authorities
     attr_accessor :certificate_authorities
 
+    # Configure if RFC 7033 WebFinger discovery should fallback to http if https fails (default: +false+)
```

Comment:
Nit: I would change this to "Configure if  ~RFC 7033~ WebFinger discovery should fallback to http if https fails", since strictly saying falling back to http is not compliant with RFC 7033. To make it clear that we talk only about the "new" WebFinger we can add another comment line saying:

```ruby
# This setting only applies to the WebFinger route from RFC 7033 `/.well-known/webfinger`. Legacy WebFinger flow unconditionally falls back to http.
```